src/lib.rs

```diff
@@ -38,3 +51,98 @@ impl SolanaRoot {
 
 pub mod kubernetes;
 pub mod release;
+
+static PACKAGE: Emoji = Emoji("📦 ", "");
+static TRUCK: Emoji = Emoji("🚚 ", "");
+
+/// Creates a new process bar for processing that will take an unknown amount of time
+pub fn new_spinner_progress_bar() -> ProgressBar {
+    let progress_bar = ProgressBar::new(42);
+    progress_bar.set_style(
+        ProgressStyle::default_spinner()
+            .template("{spinner:.green} {wide_msg}")
+            .expect("ProgresStyle::template direct input to be correct"),
+    );
+    progress_bar.enable_steady_tick(Duration::from_millis(100));
+    progress_bar
+}
+
+pub fn cat_file(path: &PathBuf) -> std::io::Result<()> {
+    let mut file = File::open(path)?;
+    let mut contents = String::new();
+    file.read_to_string(&mut contents)?;
+    info!("version.yml:\n{}", contents);
```

Comment:
nit: how about printing the filename?
```suggestion
    info!("{}:\n{}", path.file_name(), contents);
```